tests/Contacts/PhoneNumberTest.m

```diff
@@ -15,6 +11,13 @@ @interface PhoneNumberTest : XCTestCase
 
 @implementation PhoneNumberTest
 
+-(void)testE164 {
+    XCTAssertEqualObjects(@"+19025555555", [[PhoneNumber tryParsePhoneNumberFromText:@"+1 (902) 555-5555" fromRegion:@"US"] toE164]);
+    XCTAssertEqualObjects(@"+19025555555", [[PhoneNumber tryParsePhoneNumberFromText:@"1 (902) 555-5555" fromRegion:@"US"] toE164]);
+    XCTAssertEqualObjects(@"+19025555555", [[PhoneNumber tryParsePhoneNumberFromText:@"1-902-555-5555" fromRegion:@"US"] toE164]);
+    XCTAssertEqualObjects(@"+19025555555", [[PhoneNumber tryParsePhoneNumberFromText:@"1-902-５５５-5555" fromRegion:@"US"] toE164]);
```

Comment:
Would it also be worthwhile to check other input:

* Non-US calling codes.
* Phone numbers missing a calling code.
* Phone numbers with a calling code but without a +.
* Empty input.
* Incomplete (short) input.
* Overlong input.
* Bad input.